ld_script.txt

```diff
@@ -495,6 +496,7 @@ SECTIONS {
         src/trig.o(.rodata);
         src/util.o(.rodata);
         src/daycare.o(.rodata);
+        src/debug.o(.rodata);
```

Comment:
Not necessary as per https://github.com/pret/pokeemerald/pull/1738

Reply:
removed